sys-auth/seatd/files/seatd.initd

```diff
@@ -0,0 +1,4 @@
+#!/sbin/openrc-run
```

Comment:
It seems like you've added `initd` file, but you're not installing it in ebuild?

Reply:
Oh, nice catch, looks like I screwed up somewhere as I'm sure I wrote it at some point.